libs/logging/types-rust/src/types.rs

```diff
@@ -44,10 +44,10 @@ pub enum User {
 #[serde(deny_unknown_fields)]
 pub struct Log {
     pub source: String,
-    #[serde(rename = "event-id")]
+    #[serde(rename = "eventId")]
```

Comment:
Rather than doing this, you should apply `#[serde(rename_all = "camelCase")]` to the `Log` struct.